deploy/helm/sumologic/templates/setup-job.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.sumologic.setupEnabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "sumologic.fullname" . }}-init
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    app: {{ template "sumologic.labels.app" . }}
+    {{ template "sumologic.labels.common" . }}
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: setup
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/usr/bin/setup.sh", 
+          "-n", "{{ .Release.Namespace }}", 
+          "-k", "{{ .Values.sumologic.clusterName }}", 
+          {{- if .Values.sumologic.collectorName }}
+          "-c", "{{ .Values.sumologic.collectorName }}",
+          {{- end }}
+          "-d", "false", 
+          "-y", "false", 
+          '{{ required "A valid .Values.sumologic.endpoint entry required!" .Values.sumologic.endpoint }}',
```

Comment:
What are your thoughts on instead of using the full URL, we let the customer specify the deployment name (e.g. US2)? We then do a conditional block rendering the right URL based on what they selected and can fail if they make an invalid choice. This makes it easier for people, other various open-source Sumo projects work with this way.  We have to occasionally update that as we add new deployments, but that seems minor.

Reply:
It will be more maintenance work and I think we generally try to avoid it because it's easily forgotten. What are your thoughts on this @lei-sumo @samjsong @rvmiller89 ?

Reply:
I'm fine with documenting it in our steps for now.